openfl/databases/tensor_db.py

```diff
@@ -162,7 +172,7 @@ def get_aggregated_tensor(self, tensor_key: TensorKey, collaborator_weight_dict:
                                 & (self.tensor_db['report'] == report)
                                 & (self.tensor_db['tags'] == tags)]['nparray']
         if len(raw_df) > 0:
-            return np.array(raw_df.iloc[0]), {}
+            return np.array(raw_df.iloc[0]) # , {}
```

Comment:
The commit message says "temporarily commenting out ...". Is this something we should uncomment back?

Reply:
@brandon-edwards

Reply:
After the initial commit I realized it is a change we should keep.